Doc/library/sqlite3.rst

```diff
@@ -868,21 +875,27 @@ Cursor Objects
 
    .. method:: execute(sql[, parameters])
 
-      Executes an SQL statement. Values may be bound to the statement using
+      Execute an SQL statement. Values may be bound to the statement using
```

Comment:
To note, this never seems to explicitly describe what `parameters` are or in what form they should be passed, though perhaps out of scope here.

Reply:
Lets do that in another PR.